### PR TITLE
Add duplicate tab shortcut docs

### PR DESCRIPTION
**Pull request by BlairCurrey**
## Description
Adds documentation for new shortcut

## Motivation and Context
Dependent on this PR in Files: https://github.com/files-community/Files/pull/7884

### CODE REVIEW

**Comment by BlairCurrey** [comment]
I have reviewed the failed CI checks and do not see any relation to the changes I made. I am assuming they are unrelated - correct me if I'm wrong.

**Comment by TheOnlyTails** [comment]
They are unrelated, no problem on your part.

**Comment by TheOnlyTails** [comment]
It looks like SvelteKit breaking changes are crashing this, I'll have to fix this before moving on.

**Comment by TheOnlyTails** [comment]
This is not such a major/urgent change though, so it's not that big a deal.

File: src/data/features.ts [hunk 1]
```diff
@@ -1,7 +1,7 @@
 import Cloud from "@fluentui/svg-icons/icons/cloud_24_regular.svg?raw";
 import Tag from "@fluentui/svg-icons/icons/tag_24_regular.svg?raw";
 import TabDesktop from "@fluentui/svg-icons/icons/tab_desktop_20_regular.svg?raw";
-import EyeVisible from "@fluentui/svg-icons/icons/eye_show_20_regular.svg?raw";
+import EyeVisible from "@fluentui/svg-icons/icons/eye_20_regular.svg?raw";
```
Comment (BlairCurrey) [inline src/data/features.ts:4]: not quite sure why this is showing as a change.... see that it was merged in recently here (to fix CI I believe) https://github.com/files-community/Website/commit/31b9d6f8a8dddf1fa468abf9f7bcd3a67fccae69

Not sure it's a problem but I wanted to point it out.

**Comment by BlairCurrey** [comment]
This was updated to reflect the change in the shortcut here https://github.com/files-community/Files/pull/7901, which was made in response to this request https://github.com/files-community/Files/pull/7884#discussion_r790197339

**Review by chingucoding — APPROVED** [review]
Thanks for adding the shortcut to the list (and sorry for the confusion with CI)!

**Pull request merged**